python/lsst/ts/standardscripts/maintel/base_close_loop.py

```diff
@@ -442,6 +446,11 @@ async def arun(self, checkpoint: bool = False) -> None:
             # Run the operational mode handler function.
             await self.operation_model_handlers[self.mode]()
 
+            # Compute the rotation angle
+            rotation_angle = await self.mtcs.rem.mtrotator.tel_rotation.next(
```

Comment:
just one question. The data is taken before this point here. Do you care that the rotator will have change positions? you may want to have a different strategy where you get the rotation angle before/after the data is being taken and pass an average value?

Reply:
I added the average as you suggest